src/event_form/components/calendar/form-event-calendar.component.html

```diff
@@ -14,22 +14,37 @@
 <div class="calendar-timing-info">
     <div class="panel panel-default">
         <div class="panel-body">
-            <div class="calendar-time-spans" ng-if="!calendar.weeklyRecurring">
+            <div class="calender-time-init" ng-show="calendar.showBootstrapTimeSpans && calendar.type === 'single'">
+                <div class="form-group">
+                  <label for="time-span-init-start-date">Wanneer start je evenement?</label>
+                 </div>
+                <div class="form-inline">
+                 <div class="date">
+                   <udb-form-calendar-datepicker ng-model="calendar.bootstrapDate" ng-change="calendar.delayedTimeSpanChanged()">
+                   </udb-form-calendar-datepicker>
+                 </div>
+                 <button type="button" ng-click="calendar.confirmBootstrap()" class="btn btn-primary">Bevestigen</button>
+              </div>
+            </div>
+            <div class="calendar-time-spans" ng-if="!calendar.weeklyRecurring && !calendar.showBootstrapTimeSpans">
```

Comment:
Om te vermijden dat er teveel logica in de templates zit, zou ik condities simpel houden. `calendar.showBootstrapTimeSpans` kan hier ook rekening houden met `calendar.weeklyRecurring`? Op die manier kan je in de achtergrond ook unit-testen dat deze variabele altijd correct staat.